checklist_plus/text_generation/llm.py

```
@@ -205,7 +296,7 @@ def unmask_multiple(self, texts, n_completions=1, prompt_config=cfg.config.text_
         # print(f'Total unique completions generated: {len(unique_completions)}')
         return all_results
 
-    def unmask(self, text_with_mask, n_completions=10, candidates=None, context=None, **kwargs):
+    def unmask(self, text_with_mask, n_completions=10, candidates: list[str]=None, context: str=None, examples: list[TextExample | tuple[str, str] | dict[str, str]] = None, **kwargs):
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)